README.rst

```diff
@@ -40,12 +40,28 @@ Features
 Configuration
 -------------
 
-Go to the ZMI -> portal_properties -> loginlockout_properties,
+Go to the Plone Control Panel -> LoginLockout Configuration Panel -> Configuration Settings,
 there you can changes these defaults:
 
 - allowed incorrect attempts: 3
 - reset period: 24 hours
 - whitelist_ips: [] # any origin IP is allowed
+- Fake Client IP: false
+
+Go to Plone Control Panel::
+    >>> from zope.component import getUtility
+    >>> from plone.registry.interfaces import IRegistry
+    >>> from Products.LoginLockout.interfaces import ILoginLockoutSettings
+    >>> settings = getUtility(IRegistry).forInterface(ILoginLockoutSettings, prefix="Products.LoginLockout")
+    >>> settings.fake_client_ip
```

Comment:
Whats this bit about?

Reply:
oh I see. Maybe move the documentation about where to go to change the settings closer to the browser tests that go to the settings to make it clearer. If there is a way to hide the imports that would also help.
I also prefer to use labels rather in browser actions rather than names as it makes it easier to read.

Reply:
@djay do i need any changes or action for `requested changes`?